services/management/dbaas/psmdb_cluster_service.go

```diff
@@ -94,10 +94,6 @@ func (s PSMDBClusterService) ListPSMDBClusters(ctx context.Context, req *dbaasv1
 			},
 		}
 
-		if c.Params.Paused && cluster.State == dbaasv1beta1.PSMDBClusterState_PSMDB_CLUSTER_STATE_READY {
```

Comment:
`psmdbStates()` map doesn't have paused state

Reply:
Fixed. And added pxcStates() to proper convert pxc states as well.